vendor/engines/secure_rooms/app/models/secure_rooms/card_reader.rb

```diff
@@ -0,0 +1,9 @@
+module SecureRooms
+
+  class CardReader < ActiveRecord::Base
+
+    belongs_to :secure_room, foreign_key: :product_id
```

Comment:
Will validation come in the next PR (which I'm assuming will be the crud for these)? I'm cool if so, just want to check.